test/test.js

```diff
@@ -18,7 +18,7 @@ const App = () => {
     >
       <h1>Hello World!</h1>
       <button className="btn" onClick=${e => setToggle(!toggle)}>Toggle</button>
-      <div className=${css`https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css`}>
+      <div className=${css`https://unpkg.com/tailwindcss@^1.1.4/dist/tailwind.min.css`}>
```

Comment:
This still has the `^` in the URL (which was the bit I was concerned about). We want the version to be fixed.

Reply:
Ah, forgot about that, was too focused on getting the version number right :p